worlds/ffmq/Options.py

```diff
@@ -210,7 +210,7 @@ class MapShuffleSeed(FreeText):
     """If this is a number, it will be used as a set seed number for Map, Crest, and Battlefield Reward shuffles.
```

Comment:
I'm unsure how exactly this works, so I'm wondering whether this sentence should also be updated to include Companion and Kaeli's Mom options. Though, looking at the options, I doubt Kaeli's Mom is impacted directly by this option. It could be something like this:
```suggestion
    """If this is a number, it will be used as a set seed number for Map, Crest, Battlefield Reward, and Companion shuffles.
```
Other than that potentially needing an update, this looks good to me